services/ux-backend/handlers/clientonboardingtokens/handler.go

```diff
@@ -74,10 +76,11 @@ func handlePost(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int
 			klog.Errorf("failed write data to response writer: %v", err)
 		}
 	}
+
 }
 
 func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
-	klog.Info("Only POST method should be used to send data to this endpoint /onboarding-tokens")
+	klog.Info("Only POST method should be used to send data to this endpoint /client-onboarding-tokens")
```

Comment:
You cannot change the endpoint, we need to keep the endpoint for backport compatibility